### PR TITLE
reindex-index-data: add --chunk-size arg

### PR DESCRIPTION
**Pull request by sevein**
New `--chunk-size` argument allows me to control the size of the batches in bulk indexing, which is useful when dealing with read timeouts likely because of the size of the documents that we are indexing.

### CODE REVIEW

File: tools/add-fpr-rule [hunk 1]
```diff
@@ -1,4 +1,22 @@
-#!/usr/bin/python
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+# This file is part of Archivematica.
+#
+# Copyright 2010-2013 Artefactual Systems Inc. <http://artefactual.com>
```
Comment (Hwesta) [inline tools/add-fpr-rule]: Last copyright year should be 2016

File: tools/add-fpr-rule [hunk 2]
```diff
@@ -1,4 +1,22 @@
-#!/usr/bin/python
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+# This file is part of Archivematica.
```
Comment (Hwesta) [inline tools/add-fpr-rule]: Is it? Or is dev-tools considered a separate project for legal headers like this?

File: tools/reindex-index-data [hunk 3]
```diff
@@ -62,6 +81,7 @@ def reindex(index, doc_types):
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description='Regenerate index based on the existing data to update the index types.')
     parser.add_argument('index', choices=['transfers', 'aips'], help='Index to re-generate.')
+    parser.add_argument('--chunk-size', default=50, help='Number of docs in one chunk sent to Elasticsearch.')
```
Comment (Hwesta) [inline tools/reindex-index-data]: This should include `type=int` or `args.chunk_size` will be a string when not the default.

**Comment by Hwesta** [comment]
Please update the README with the new parameter info.

File: tools/add-fpr-rule (hunk 2 above)
Comment (sevein) [inline tools/add-fpr-rule]: No idea, I'll ask Justin and send another PR if it needs to be changed.

**Comment by sevein** [comment]
Done! Please take a look! :)

File: README.md [hunk 4]
```diff
@@ -99,3 +99,4 @@ This is useful if the mapping has changed in an incompatible way but the existin
 Depending on the size of the index, a large amount of memory may be consumed.
 
 There is one parameter which specifies which index to recreate: `transfers` or `aips`.
```
Comment (Hwesta) [inline README.md]: This should be update because there's multiple parameters now and this one's required.

File: tools/add-fpr-rule (hunk 2 above)
Comment (Hwesta) [inline tools/add-fpr-rule]: Or just update this one, if it hasn't been merged yet.

**Comment by Hwesta** [comment]
:+1: 

The CR-related commits can (should?) be squashed with the original commit they're updating.

**Pull request merged**